src/controllers/ratings.controller.js

```diff
@@ -0,0 +1,73 @@
+import db from '../database/models';
+import BaseRepository from '../repository/base.repository';
+import responseGenerator from '../helpers/responseGenerator';
+
+const { Ratings, Article } = db;
+
+/** Rating Class */
+class RatingsController {
+  /**
+   * Handles User ability to rate Articles
+   * @method
+   * @async
+   * @param  {Object}   req  - express http request object
+   * @param  {Object}   res  - express http response object
+   * @return {Object}        - returns an http response object
+   * @static
+   */
+  static async rateArticles(req, res) {
+    // needed parameters
+    const { id } = req.params;
+    const userId = req.currentUser.id;
+    const { ratings } = req.body;
+
+    const articleId = id;
+
+    try {
+      const article = await BaseRepository.findOneByField(Article, {
```

Comment:
Great job so far @codexempire . However, this method (check existence of an article)  already exists as a middleware. Could you plug that and remove this part? You wouldn't need an `if` statement here anymore then